.buildkite/scripts/imageprep.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/setenv.sh
+
+DOCKER_REGISTRY = 'docker.elastic.co'
+DOCKER_REGISTRY_SECRET_PATH = "kv/ci-shared/platform-ingest/docker_registry_prod"
+
+publish_docker_image() {
+    echo "Pushing the docker image "$DOCKER_IMAGE:$DOCKER_IMAGE_TAG" to the ${DOCKER_REGISTRY} registry..."
+    dockerLogin
+    docker push "$DOCKER_IMAGE:$DOCKER_IMAGE_TAG"
+    docker logout "$DOCKER_REGISTRY"
```

Comment:
This command should be added as a bash trap , to ensure that always (even with error) those credentials are removed from the host.

Reply:
Done, please check it.
I didn't find a better way to delete credentials. 
Please let me know if you have other ideas...